docs/sources/tempo/configuration/grafana-agent/automatic-logging.md

```diff
@@ -19,15 +21,15 @@ Automatic logging provides an easy and fast way of discovering trace IDs
 through log messages. Well-formatted log lines are written to a Loki instance
 or to `stdout` for each span, root, or process that passes through the tracing
 pipeline. This allows for automatically building a mechanism for trace
-discovery. On top of that, we also get metrics from traces using Loki, and
+discovery. On top of that, you can also get metrics from traces using Loki, and
 allow quickly jumping from a log message to the trace view in Grafana.
 
 While this approach is useful, it isn't as powerful as [TraceQL]({{< relref
 "../../traceql" >}}). If you are here because you know you want to log the
 trace ID, to enable jumping from logs to traces, then read on!
 
 If you want to query the system directly, read the [TraceQL
-documentation]({{< relref "../../traceql" >}}). 
+documentation]({{< relref "../../traceql" >}}).
```

Comment:
Same question here for this relref... which applies to the other relrefs as well

Reply:
```suggestion
documentation](https://grafana.com/docs/tempo/<TEMPO_VERSION>/traceql).
```